loopback.go

```diff
@@ -360,9 +362,15 @@ func (n *Node) Create(
 
 	h := &Handle{fs: n.fs, f: f, reopener: opener}
 
-	node := &Node{realPath: filepath.Join(n.realPath, req.Name),
-		isDir: req.Mode.IsDir(), fs: n.fs}
-	resp.Handle = n.fs.newHandle(h)
+	node := &Node{
+		realPath: filepath.Join(n.realPath, req.Name),
+		isDir:    req.Mode.IsDir(),
+		fs:       n.fs,
+	}
+	node.rememberHandle(h)
```

Comment:
Correct; `resp.Handle` is a HandleID, which gets overridden by basil anyway, so there's no need to generate them here